packages/backend/src/database/migrations/20241021110921_add-timezone-to-scheduler.ts

```diff
@@ -0,0 +1,17 @@
+import { Knex } from 'knex';
+
+const SchedulerTableName = 'scheduler';
+
+export async function up(knex: Knex): Promise<void> {
+    if (await knex.schema.hasTable(SchedulerTableName)) {
+        await knex.schema.alterTable(SchedulerTableName, (table) => {
+            table.string('timezone').defaultTo('UTC').notNullable();
```

Comment:
if we do this https://lightdash.slack.com/archives/C02GQKJK84Q/p1729579976689549?thread_ts=1729517981.947559&cid=C02GQKJK84Q
, this should be nullable()